scripts/jenkins/ardana/ansible/init.yml

```diff
@@ -151,6 +151,26 @@
     tags:
       - create-ardana-input-model
 
+  - name: install patch
+    set_fact:
+        patches: {{ tempest_run_filter}}
+
+  - name: install patch
+    shell: |
+        zypper -n in patch
+    # https://gist.github.com/marcusphi/6791404
+    when: patches is defined and (not patches|trim == '')
+
+  - name: patchit
+    become: true
+    become_user: ardana
+    shell: |
+        curl -k '{{ item }}' | patch -p1
+    args:
+      chdir: /var/lib/ardana//openstack/ardana/ansible
+    with_items: "{{ patches.split(',')}}"
+    when: patches is defined and (not patches|trim == '')
```

Comment:
Maybe you can use this:
http://docs.ansible.com/ansible/latest/patch_module.html

Reply:
Thanks, I looked, but the module needs the file to be already present, either on host or remote machine.

Reply:
So download it first via `get_url`?

Reply:
Also for something like this `https://gerrit.suse.provo.cloud/changes/2985/revisions/current/patch?zip | gunzip`